config/apply_update.sh

```diff
@@ -73,35 +73,35 @@ if [ "$PHONE_DEV" == "" ]; then
 	exit 1
 fi
 
-PHONE_MOUNT=`df -P | grep $PHONE_DEV | awk '{print $6}'`
+PHONE_MOUNT=`df -P | grep "$PHONE_DEV" | awk '{print $6}'`
 
 if [ "$PHONE_MOUNT" == "" ]; then
-	echo dev: $PHONE_DEV not mounted, do that yorself and re-run
+	echo "dev: $PHONE_DEV not mounted, do that yorself and re-run"
 	exit 1
 fi
 
-if [ ! -e $UPDATE_FILE ]; then
+if [ ! -e "$UPDATE_FILE" ]; then
 	echo "Update file $UPDATE_FILE does not exist"
 	exit 1
 fi
 
-rm -rf $TMPDIR
-mkdir $TMPDIR
-tar -C $TMPDIR -xpf $UPDATE_FILE
+rm -rf "$TMPDIR"
+mkdir "$TMPDIR"
+tar -C "$TMPDIR" -xpf "$UPDATE_FILE"
 
 if [ $CLEAN_PHONE == 1 ]; then
 	echo "Clean phone requested"
-	rm -rf $PHONE_MOUNT/current
-	mkdir -p $PHONE_MOUNT/current
+	rm -rf "$PHONE_MOUNT"/current
+	mkdir -p "$PHONE_MOUNT"/current
 	sync
 fi
 
 echo "Copyind data"
-cp $TMPDIR/boot.bin $PHONE_MOUNT/current/
-cp $TMPDIR/Luts.bin $PHONE_MOUNT/current/
-cp $TMPDIR/country-codes.db $PHONE_MOUNT/current/
-cp $TMPDIR/version.json $PHONE_MOUNT/current/
-cp -r $TMPDIR/assets $PHONE_MOUNT/current/
+cp "$TMPDIR"/boot.bin "$PHONE_MOUNT"/current/
```

Comment:
Just a note. A whole expresion could be quoted. (e.g. "$TMPDIR/boot.bin").
In my opinion it is a more readable form.

Reply:
I've updated the commit to reflect the desired semantic path quoting.